provisionerd/proto/provisionerd.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.30.0
-// 	protoc        v4.23.3
+// 	protoc-gen-go v1.34.2
+// 	protoc        v4.25.3
```

Comment:
was this intentional?

Reply:
Nope! Good catch.

Reply:
Actually, we don't lock these versions.
In my dogfood workspace running nix, i've got:

```bash
$ protoc-gen-go --version
protoc-gen-go v1.34.2
$ protoc --version
libprotoc 25.3

$ which protoc protoc-gen-go
/nix/var/nix/profiles/default/bin/protoc
/nix/var/nix/profiles/default/bin/protoc-gen-go
```

Reply:
I think CI uses the 'regular' dogfood image as opposed to 'nix'. This probably isn't a huge issue tbh but just wanted to double-check.

Reply:
I would beg to differ.

If our dev environments' and our CI environment's dependencies are not in lock-step, this could lead not only to CI frustrations like the one above but to something more pernicious. We're either getting false-positives or false-negatives: any alignment is temporary/incidental, and we have no source of truth.

Reply:
Added https://github.com/coder/coder/issues/14343 as a follow-up.